src/create_mtc_pipeline_planner.cpp

```diff
@@ -31,38 +35,61 @@ BT::NodeStatus CreateMTCPipelinePlanner::tick()
   double goal_joint_tolerance;
   double max_velocity_scaling_factor;
   double max_acceleration_scaling_factor;
+  double goal_orientation_tolerance;
+  double goal_position_tolerance;
+  bool display_motion_plans;
+  bool publish_planning_requests;
+  uint num_planning_attemps;
 
-  if(!getInput(kPortPipelineID, pipeline_id) ||
-     !getInput(kPortPlannerID, planner_id) ||
-     !getInput(kPortMaxVelocityScalingFactor, max_velocity_scaling_factor) ||
-     !getInput(kPortMaxAccelerationScalingFactor, max_acceleration_scaling_factor))
+  //Mandatory inputs
+  if(!getInput(kPortPipelineID, pipeline_id))  //ex : "ompl"
     return NodeStatus::FAILURE;
-
-  getInput(kPortGoalJointTolerance, goal_joint_tolerance);  //optional
-
-  // Build solver
+  if(!getInput(kPortPlannerID, planner_id))  //ex : "request_adapters"
+    return NodeStatus::FAILURE;
+  if(!getInput(kPortMaxVelocityScalingFactor, max_velocity_scaling_factor))
+    return NodeStatus::FAILURE;
+  if(!getInput(kPortMaxAccelerationScalingFactor, max_acceleration_scaling_factor))
+    return NodeStatus::FAILURE;
+  //Optionnal inputs
+  getInput(kPortGoalJointTolerance, goal_joint_tolerance);
+  getInput(kPortGoalPositionTolerance, goal_position_tolerance);
+  getInput(kPortGoalOrientationTolerance, goal_orientation_tolerance);
+  getInput(kPortDisplayMotionPlans, display_motion_plans);
+  getInput(kPortPublishPlanningRequests, publish_planning_requests);
+  getInput(kPortNumPlanningAttempts, num_planning_attemps);
```

Comment:
Remove comments `//ex : ...`, the second one is not valid.

For the time being, let's make all inputs mandatory. Rework all the getInputs like the following:
```cpp
if(!getInput(kPortPipelineID, pipeline_id) ||
   !getInput(kPortPlannerID, planner_id) ||
   ...
   !getInput(kPortNumPlanningAttempts, num_planning_attemps))
      return NodeStatus::FAILURE;
```